crates/nu-cmd-lang/src/core_commands/const_.rs

```diff
@@ -46,6 +46,9 @@ impl Command for Const {
         call: &Call,
         _input: PipelineData,
     ) -> Result<PipelineData, ShellError> {
+        // This is compiled specially by the IR compiler. The code here is never used in when
```

Comment:
seems like grammar was wrong? i wouldn't really care but this same statement is all over the place. 😄 
```suggestion
        // This is compiled specially by the IR compiler. The code here is never used when
```

Reply:
You're right, I don't know how I missed that 😅 

I'll find and replace it